pkg/core/native/native_neo.go

```diff
@@ -323,6 +346,11 @@ func (n *NEO) PostPersist(ic *interop.Context) error {
 		n.gasPerBlock.Store(gr)
 		n.gasPerBlockChanged.Store(false)
 	}
+	if n.registerPriceChanged.Load() {
+		p := n.getRegisterPriceInternal(ic.DAO)
```

Comment:
`n.getRegisterPriceInternal` also checks for `n.registerPriceChanged.Load`.

Reply:
Fixed.